README.md

```diff
@@ -92,6 +92,28 @@ There are many other options to explore. You can see them all by running:
     pdoc --help
 
 
+Submodule loading
+-----------------
+
+`pdoc` uses idiomatic Python when loading your modules. Therefore, for `pdoc`
+to find any submodules of the input module you specify on the command line,
+those modules must be available through Python's ordinary module loading process.
+
+This is not a problem for globally installed modules like `sys`, but can be
+a problem for your own sub-modules depending on how you have installed them.
```

Comment:
Could you add a clarification note that says `pdoc` works well with `virtualenv`? (Which is what "idiomatic Python" means to me.) Namely, if you're documenting your project, and you're using a `virtualenv`, then installing `pdoc` into that `virtualenv` and using it should work perfectly.